lib/profiler/get_tick.h

```diff
@@ -67,6 +67,15 @@ get_tick(void)
     return regs;
 }
 
+#elif defined(HAVE_CLOCK_GETTIME_NSEC_NP)
+#include <time.h>
+
+static __inline hrtime_t
+get_tick(void)
+{
+    return clock_gettime_nsec_np (CLOCK_UPTIME_RAW);
```

Comment:
Nitpick: Squid usually does not separate the function name from its arguments:

```suggestion
    return clock_gettime_nsec_np(CLOCK_UPTIME_RAW);
```

Reply:
sure ... btw all of these style changes can be automated with clang format setting possibly ?

Reply:
Yes, most can be. We already auto-format with AStyle, but not every detail like this is covered by the existing formatting rules (unfortunately) because, in part, AStyle is not powerful and careful enough. An attempt to upgrade to clang-based formatting has stalled at #603.